stdlib/public/core/AssertCommon.swift

```diff
@@ -21,6 +21,7 @@ import SwiftShims
 
 @_inlineable // FIXME(sil-serialize-all)
 @_transparent
+@compilerEvaluable
```

Comment:
Not important for this patch of course, but I'd be very curious to know what functions are marked @_transparent that do not end up being marked as compilerEvaluable...